e2e/atlas_whitelist_test.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// +build e2e
+
+package e2e
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	"github.com/mongodb/go-client-mongodb-atlas/mongodbatlas"
+)
+
+func TestAtlasWhitelist(t *testing.T) {
+	cliPath, err := filepath.Abs("../bin/mcli")
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+	_, err = os.Stat(cliPath)
+
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	atlasEntity := "atlas"
+	whitelistEntity := "whitelist"
+	entry := "77.107.234.162"
```

Comment:
[q] Why this IP and not just a testing one? `whois` shows this as some IP in Ireland - not sure if we should have that in the E2E tests if we don't need it.

Reply:
Atlas has validations, since this is for realsy I didn't want it to fail because of the validations

Reply:
changed to 192.168.0.1, let's see how that goes

Reply:
Cool. I'd probably have tried google (8.8.8.8) or something in that case - ip's should go away but I have no idea what would happen to the test if it did.